src/parseExpensiMark.ts

```diff
@@ -252,13 +252,16 @@ function parseExpensiMark(markdown: string): MarkdownRange[] {
     );
     return [];
   }
+  const sortedRanges = sortRanges(ranges);
+  let markdownRanges = sortedRanges;
+  if (Platform.OS === 'android') {
```

Comment:
Since `parseExpensiMark` is a worklet, let's extract `Platform.OS` outside to avoid capturing whole `Platform` object in worklet closure.
```ts
const isAndroid = Platform.OS === 'android';

function parseExpensiMark(...) {
  if (isAndroid) { /* ... */ }
}
```